cloud_run_service_image_processing/main.tf

```diff
@@ -0,0 +1,110 @@
+provider "google" {
+  project = "muncus-dev2"
+}
+# [START cloudrun_service_image_processing_datasources]
+data "google_project" "project" {}
+
+data "google_storage_project_service_account" "gcs_account" {}
+# [END cloudrun_service_image_processing_datasources]
+
+# [START cloudrun_service_image_processing_inputbucket]
+# Input Bucket
+resource "google_storage_bucket" "imageproc_input" {
+ name     = "imageproc-input-bucket"
```

Comment:
Lets add a random prefix to prevent collisions
https://github.com/terraform-google-modules/terraform-docs-samples/blob/d143b3bba904b64951992dffbd20f539a2764078/cloudfunctions2_basic/main.tf#L7